site/cni-plugin.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Integrating Kubernetes via the CNI Plugin
+title: Weave CNI Plugin
```

Comment:
IMO that's putting the cart before the horse. "Configuring the CNI Plugin" is not something a user wants to do. It's a means to and end. And the end is using Weave Net with K8s or Mesos.

Also, note the two TOC entries just above this one:
- Integrating Docker via the API Proxy
- Integrating Docker via the Network Plugin

The title I suggested follows the same pattern.